crates/controller/src/lib.rs

```diff
@@ -76,8 +76,6 @@ impl PDController {
         max_inflation.to_uint().ok_or(Error::MaxInflationOverflow)
     }
 
-    // TODO: could possibly use I256 instead of Dec here (need to account for
-    // negative vals)
```

Comment:
@brentstone I think this is ok, right?

Reply:
I removed this TODO, don't think we gain much by using `I256` instead of `Dec`